tests/_support/Helper/AsyncApiHelper.php

```diff
@@ -389,4 +387,72 @@ public function getAsyncApiBuilderConsoleMock(): BuildCodeFromAsyncApiConsole
 
         return $buildFromAsyncApiConsole;
     }
+
+    /**
+     * @return \Generated\Shared\Transfer\AsyncApiMessageTransfer
+     */
+    public function haveAsyncApiMessagePropertyRequestTransfer(): AsyncApiMessageTransfer
+    {
+        $asyncApiChannelTransfer = new AsyncApiChannelTransfer();
+        $asyncApiChannelTransfer->setName(static::CHANNEL_NAME);
+
+        $asyncApiMessageTransfer = new AsyncApiMessageTransfer();
+        $asyncApiMessageTransfer
+            ->setChannel($asyncApiChannelTransfer)
+            ->setName('message')
+            ->setProperty(['firstName:string:required', 'lastName:string', 'phoneNumber:int:required'])
+            ->setContentType('object');
```

Comment:
could you use data builders to generate transfers
https://docs.spryker.com/docs/scos/dev/guidelines/testing-guidelines/data-builders.html

Reply:
Next time we can introduce databuilders. Not now.